addons/binding/org.openhab.binding.fronius/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="fronius" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="powerinverter">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="bridge" />
+        </supported-bridge-type-refs>	
+		<label>Fronius Symo inverter</label>
+		<description>Fronius Symo power inverter</description>
+
+		<channels>
+			<channel id="inverterdatachannelpac" typeId="pac" />
+			<channel id="inverterdatachanneldayenergy" typeId="day_energy" />
+			<channel id="inverterdatachanneltotal" typeId="total_energy" />
+			<channel id="inverterdatachannelyear" typeId="year_energy" />
+			<channel id="inverterdatachannelfac" typeId="fac" />
+			<channel id="inverterdatachanneliac" typeId="iac" />
+			<channel id="inverterdatachannelidc" typeId="idc" />
+			<channel id="inverterdatachanneluac" typeId="uac" />
+			<channel id="inverterdatachanneludc" typeId="udc" />
+			<channel id="powerflowchannelpgrid" typeId="pGrid" />
+			<channel id="powerflowchannelpload" typeId="pLoad" />
+			<channel id="powerflowchannelpakku" typeId="pAkku" />
+		</channels>
+
+		<config-description>
+			<parameter name="deviceId" type="integer">
+				<label>DeviceId</label>
+				<description>Specific device identifier</description>
+				<default>1</default>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="day_energy">
+		<item-type>Number</item-type>
+		<label>Day Energy</label>
+		<description>Energy generated on current day</description>
+		<state pattern="%.2f kWh" readOnly="true"></state>
+	</channel-type>
+	<channel-type id="pac">
+		<item-type>Number</item-type>
+		<label>AC powery</label>
+		<description>AC power</description>
+		<state pattern="%f W" readOnly="true"></state>
+	</channel-type>
+	<channel-type id="total_energy">
+		<item-type>Number</item-type>
+		<label>Total Energy</label>
+		<description>Energy generated overall</description>
+		<state pattern="%.4f MWh" readOnly="true"></state>

```

Comment:
Yes, that's the total energy from the beginning, you're fast in a MWh range.

Reply:
I know, but I nonetheless often rather see "60.000 kWh" or similar. But if you prefer MWh, I can live with that. Maybe just consider to change it to "%.3f MWh", as I think will be much easier understandable (as easier to be transformed to kWh) if it is "60,000 MWh" instead of "60,0000 MWh".